fighthealthinsurance/fax_actor.py

```
@@ -5,11 +5,14 @@
 import time
 import asyncio
 
-from django.core.mail import EmailMultiAlternatives
+from django.core.mail import EmailMultiAlternatives, BadHeaderError
```

Ruff (0.8.2)

`django.core.mail.BadHeaderError` imported but unused

Remove unused import: `django.core.mail.BadHeaderError`

(F401)